app/src/main/java/ch/epfl/favo/chat/ViewHolder/MessageViewHolder.java

```diff
@@ -46,12 +55,23 @@
   public void bind(@NonNull Message textMessage) {
     FirebaseUser currentUser = DependencyFactory.getCurrentFirebaseUser();
     setIsSender(currentUser != null && textMessage.getUid().equals(currentUser.getUid()));
-    mNameField.setText(textMessage.getName());
+    userId = textMessage.getUid();
+    SpannableString name = new SpannableString(textMessage.getName());;
+    name.setSpan(new UnderlineSpan(), 0, name.length(), 0);
+    mNameField.setText( name.subSequence(0, Math.min(name.length(), 10)));
```

Comment:
I don't know how to wrap them. What does it mean?

Reply:
We can use the "..." when the name is too long, this can be done easily on the xml attributes for each text view with:

android:ellipsize="end" 
android:maxLines="1"

Reply:
great, thanks!